### PR TITLE
Feature movefunctionality

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: app/src/test/java/src/ShapeITest.java [hunk 1]
```diff
@@ -8,9 +8,30 @@
...
     public void draw_DrawingShapeI_ArrayOfNumbers() {
-        ShapeI shapeI = new ShapeI();
+        ShapeI shapeI = new ShapeI(1, 1);
         int[][] actual = shapeI.draw();
         int[][] expected = {{ 1, 1, 1, 1}};
         assertArrayEquals(expected, actual);
     }
+
+    @Test
+    public void itShouldMoveRight() {
```
Comment (jpgonzales27) [inline app/src/test/java/src/ShapeITest.java]: joel the names of the tests are not as we define them

Comment (JRojas999) [reply app/src/test/java/src/ShapeITest.java]: You're right. I already changed them.

Comment (jpgonzales27) [reply app/src/test/java/src/ShapeITest.java]: Yes now they are fine thanks Joel

**Review by jpgonzales27 — APPROVED** [review]

File: app/src/main/java/src/Shape.java [hunk 2]
```diff
@@ -4,27 +4,58 @@
...
+     * In order to represent to move right
+     */
     @Override
     public void moveRight() {
-
+        yPosition += 1;
     }
 
+    /**
+     * Adds one to yPosition.
```
Comment (Raymundogs) [inline app/src/main/java/src/Shape.java]: Description on both methods is very similar. 
Maybe you could try something like:
Increase yposition by one
Decrease yposition by one

File: app/src/main/java/src/ShapeL.java [hunk 3]
```diff
@@ -4,6 +4,10 @@
     private static final int ROWS = 2;
     private static final int COLUMNS = 3;
 
+    public ShapeL(int xPos, int yPos) {
+        super(xPos, yPos);
+    }
+
     /**
      * Draws the shape L into an array.
      * @return An array of int with of the form of figure.
```
Comment (Raymundogs) [inline app/src/main/java/src/ShapeL.java]: remove the "of", just An array of ints with the form of a figure

Comment (JRojas999) [reply app/src/main/java/src/ShapeL.java]: Ok, It's done

File: app/src/main/java/src/Shape.java (hunk 2 above)
Comment (JRojas999) [reply app/src/main/java/src/Shape.java]: Thanks for suggestions.

**Pull request merged**